app/views/evaluation_forms/_form.html.erb

```diff
@@ -192,7 +168,7 @@
       </p>
 
       <div class="usa-label text-bold font-sans-sm">Evaluation Start Date:</div>
-      <div class="usa-hint" data-evaluation-form-target="startDate">mm/dd/yyyy</div>
+      <div class="usa-hint"><%= @phase.challenge.end_date.strftime('%m/%d/%Y') %></div>
```

Comment:
this should be the phase end_date instead of the challenge
```suggestion
      <div class="usa-hint"><%= @phase.end_date.strftime('%m/%d/%Y') %></div>
```